jax/experimental/host_callback.py

```diff
@@ -560,6 +564,15 @@ def power3_with_cotangents(x):
         'Has no effect on TPU, since only the outfeed mechanism is implemented.'
     )
 )
+_HOST_CALLBACK_LEGACY = config.DEFINE_bool(
+    'jax_host_callback_legacy',
+    config.bool_env('JAX_HOST_CALLBACK_LEGACY', False),
```

Comment:
The default value will be `True` when submitted. Using `False` for now for testing.